practice/vue-app/src/types/validation.ts

```diff
@@ -0,0 +1 @@
+export type TResultValidation = 'Valid' | 'Invalid'
```

Comment:
Skąd pomysł na ten prefix "T"?

Reply:
Kiedyś słyszałem o takiej konwencji prefixów (T - type, I - interface).  Skłaniasz się ku stwierdzeniu, że jest to neipotrzebne? Z tego co zrobiłem reaserch, to faktycznie nie te prefixy nie są pożądane xd